providers.cpp

```diff
@@ -875,12 +875,14 @@ bool MathItem::compare(const QRegExp &regExp) const
             {
                 try
                 {
+                    mParser->SetArgSep(';'); // we dont use args, we can set it to something not equal ','
```

Comment:
This is not needed to set every time.... we just need to set the localization values in initialization time.

Reply:
See https://github.com/lxqt/lxqt/issues/1672#issuecomment-463915373

Moreover, I'm not sure we could set the arg sep so. What if it's used a math function?

Reply:
its called bad luck

Reply:
If there is a math function with multiple argument that can be used, it's called a bug in muparser.

Reply:
only if one will not use the ';' - which is common for lists - our alternatives are not really better for now - it would mean a full and complete parsing

Reply:
I'm not good at this. I've lived with math but, in comparison, have come to coding only "recently". Math has universal notations (yes, some countries use `curl` instead of `rot` in vector analysis but that's not the rule); computer math seems different.

Doing it in the simplest way seems like a good idea. Set the argument separator and get rid of the problem!

Reply:
and we should use @palinek's approach and set it in the right™ place, not everytime the input is parsed.